scripts/just-task.js

```diff
@@ -0,0 +1,32 @@
+// @ts-check
+
+const path = require('path');
+const { task, series, parallel, condition, option, argv } = require('just-task');
+
+require('./just-tasks/ts');
```

Comment:
I can't help but wonder if it might be just a bit cleaner to make all those other files not inherently function off side-effects .. (so `require('./just-tasks/ts')()` instead. This is more typical CommonJS code right?)

Reply:
I can try that out in my next PR. This is just an idea of how we might export tasks. I think you're right, and I might have a bunch of functions that require in the right tasks as rigs then.